server/etc/pulp/server.conf

```diff
@@ -318,6 +318,9 @@
 #
 # login_method: Select the SASL login method used to connect to the broker. This should be left
 #     unset except in special cases such as SSL client certificate authentication.
+#
+# worker_failover_time: The maximum time a worker will run for before being considered dead and
```

Comment:
It should probably be named something other than ```worker_failover_time``` since it doesn't apply only when HA is being used.  

Maybe ```worker_expiration_time```.  I don't care too much about the name though so if you prefer it this way that's fine with me.

Reply:
What about a name like `worker_timeout` or `worker_missing_time` or `worker_missing_timeout` ?

Reply:
The `killed in most cases` makes it sound like we're actively killing the process which we don't do. I see you meant though. I'm also wondering about a little bit of guidance here too. Here's an idea, feel free to edit wholly/rewrite:

`The amount of time (in seconds) before considering a worker as missing. If Pulp's mongo database has slow I/O, then setting a higher number may resolve issues where workers are going missing incorrectly. Defaults to 30.`

Reply:
+1 for ```worker_timeout``` or ```worker_missing_timeout```